.azure-pipelines/LinuxAnalyzerTest.yml

```diff
@@ -63,9 +63,9 @@ jobs:
   # Gradle build cache, see: https://docs.microsoft.com/en-us/azure/devops/pipelines/release/caching?view=azure-devops
   - task: Cache@2
     inputs:
-      key: '"$(Agent.OS)" | analyzer-test | gradle-caches | v3 | gradle.properties, settings.gradle.kts, **/build.gradle.kts'
+      key: '"$(Agent.OS)" | analyzer-test | gradle-caches | v2 | gradle.properties, settings.gradle.kts, **/build.gradle.kts'
```

Comment:
I'd hope that we can drop this commit after another rebase.

Reply:
I checked the merged PRs from the last days and none seemed to specifically make any changes here. Is there a reason why you think this can be removed? (I'd be more than happy to)

Reply:
I simply think this is just a band aim which *might* help *sometimes*, but it does not fix the root cause. We already did bump the cache version in the past to work around the issue, and while it seemed to have fixed the issue, it apparently did not (as you're still seeing the issue). So instead of bumping the cache version over and over again, I believe we can as well merge despite the test failure and try to address the root cause in parallel.

Reply:
I dropped the commit for now. I can always add it again, if it doesn't work :)